### DISCUSSION TITLE
Hide zero values in time series panel legend

### DISCUSSION THREAD

**Discussion by jesjimher** [grafana/grafana#42140] [Dashboards, Panels & Visualizations]
In Grafana 7.x, Graph panel allowed hiding zero values in legend. Time series panel, while being much cooler in most regards, doesn't have this option. It would be nice having it also in Timeseries, not just for feature parity, but becasue it's very useful for uncluttering legends when there's a lot of series, but just a few of them have values.

Thanks in advance

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
